### PR TITLE
Seasonal Address Test Updates

### PR DESCRIPTION
**Pull request by davidjray**
# Critical Changes

# Changes
- Seasonal Address test adjustments
# Issues Closed

### CODE REVIEW

File: src/classes/ADDR_Seasonal_TEST.cls [hunk 1]
```diff
@@ -744,28 +744,32 @@ private with sharing class ADDR_Seasonal_TEST {
...
             accAddrs2[i].Default_Address__c = false;
-            accAddrs2[i].Seasonal_Start_Month__c = String.valueOf(system.today().addMonths(-1).month());
-            accAddrs2[i].Seasonal_Start_Day__c = '28';
-            accAddrs2[i].Seasonal_End_Month__c = String.valueOf(system.today().month());
+            accAddrs2[i].Seasonal_Start_Month__c = String.valueOf(system.today().addDays(-31).month());
+            accAddrs2[i].Seasonal_Start_Day__c = String.valueOf(system.today().addDays(-31).day());
+            accAddrs2[i].Seasonal_Start_Year__c = system.today().addDays(-31).year();
+            accAddrs2[i].Seasonal_End_Month__c = String.valueOf(system.today().addDays(1).month());
             accAddrs2[i].Seasonal_End_Day__c = String.valueOf(system.today().addDays(1).day());
+            accAddrs2[i].Seasonal_End_Year__c = system.today().addDays(1).year();
```
Comment (ceiroa) [inline src/classes/ADDR_Seasonal_TEST.cls]: We should not use Seasonal_Start_Year__c and Seasonal_End_Year__c, as this changes the intent of the test. We have other tests that include the year, but those are for addresses that aren't seasonal (in the sense that they don't repeat every year).

File: src/classes/ADDR_Seasonal_TEST.cls [hunk 2]
```diff
@@ -744,28 +744,32 @@ private with sharing class ADDR_Seasonal_TEST {
...
             conAddrs2[i].Default_Address__c = false;
-            conAddrs2[i].Seasonal_Start_Month__c = String.valueOf(system.today().addMonths(-1).month());
-            conAddrs2[i].Seasonal_Start_Day__c = '28';
-            conAddrs2[i].Seasonal_End_Month__c = String.valueOf(system.today().month());
+            conAddrs2[i].Seasonal_Start_Month__c = String.valueOf(system.today().addDays(-31).month());
+            conAddrs2[i].Seasonal_Start_Day__c = String.valueOf(system.today().addDays(-31).day());
+            conAddrs2[i].Seasonal_Start_Year__c = system.today().addDays(-31).year();
+            conAddrs2[i].Seasonal_End_Month__c = String.valueOf(system.today().addDays(1).month());
             conAddrs2[i].Seasonal_End_Day__c = String.valueOf(system.today().addDays(1).day());
+            conAddrs2[i].Seasonal_End_Year__c = system.today().addDays(1).year();
```
Comment (ceiroa) [inline src/classes/ADDR_Seasonal_TEST.cls]: We should not use Seasonal_Start_Year__c and Seasonal_End_Year__c, as this changes the intent of the test. We have other tests that include the year, but those are for addresses that aren't seasonal (in the sense that they don't repeat every year).

File: src/classes/ADDR_Seasonal_TEST.cls (hunk 1 above)
Comment (davidjray) [inline src/classes/ADDR_Seasonal_TEST.cls]: Good call. I wanted to make sure the year was not part of the problem.

File: src/classes/ADDR_Seasonal_TEST.cls (hunk 2 above)
Comment (davidjray) [inline src/classes/ADDR_Seasonal_TEST.cls]: Good call. I wanted to make sure the year was not part of the problem.

**Pull request merged**